lib/run.js

```diff
@@ -66,9 +66,12 @@ function setResult(policyResults) {
         if (failedCount > 0) {
             core.setFailed(`Found '${failedCount}' failure(s) while deploying policies.`);
         }
-        else {
+        else if (policyResults.length > 0) {
             core.info(`All policies deployed successfully. Created/updated '${policyResults.length}' definitions/assignments.`);
         }
+        else {
+            core.warning(`Did not find any policies to update. Please ensure that policy definition files are named policy.json. This can also happen if there is no change in policies and '${Inputs.INPUT_MODE}' is not set to '${Inputs.MODE_COMPLETE}'.`);
```

Comment:
> Does 'enforce' needs to be added in action.yml?

Yes @rgsubh. Thanks for pointing out.

Reply:
Agree we can make the message more generic rather than posting our the policy.json. I think considering we know what the mode was set as, we can print more appropriate message? when mode is Complete : Mode was set complete but no files found for this patten., Mode is Incremental either: No files found for the patter  or the file was not updated. Thoughts?

Reply:
Using `mode` is a great suggestion. I am thinking of keeping `policy.json`/`policyset.json` in the msg.

`mode == complete`:
Did not find any policies to create/update. No policy files match the given patterns.
If you have policy definitions or policy initiatives, please ensure that the files are named 'policy.json' and 'policyset.json' respectively.

`mode == incremental`:
Did not find any policies to create/update. No policy files match the given patterns or no changes were detected.
If you have policy definitions or policy initiatives, please ensure that the files are named 'policy.json' and 'policyset.json' respectively.

Reply:
Updated!